packages/ocean-react/src/Button/Button.tsx

```diff
@@ -16,7 +16,7 @@ export type ButtonProps<P extends React.ElementType = 'button'> = {
      * The variant to use.
      * @default 'primary'
      */
-    variant?: 'primary' | 'secondary' | 'text' | 'inverse';
+    variant?: 'primary' | 'secondary' | 'text' | 'inverse' | 'critical';
```

Comment:
@julianasouza-s @edu2711 

There is another critical variant. The `textCritical` as we see here:
https://www.figma.com/file/PgLv9k22CHHZZ867HwN65y/%E2%9D%96-Ocean-DS-%2F-Core-Components?node-id=2856%3A24689

if all current variations (primary, secondary, text, inverse) have a critical state, then we should consider creating a new property to represent that state between variations, like "critical={true|false}". Example: https://ant.design/components/button/#components-button-demo-danger

Otherwise, I suggest this change:
```suggestion
    variant?: 'primary' | 'secondary' | 'text' | 'inverse' | 'primaryCritical';
```

Reply:
I believe we could go for the simple route this time and add the `primaryCritical` as a variant, if we need other options aside from the `primaryCritical` and `textCritical` we should create the critical property!

Reply:
Awesome! Go ahead: `primaryCritical` == `.ods-btn--primary-critical`

Reply:
I agree with Edu. I'm not sure we'll need the critical variant for the Secondary Button ;)